### PR TITLE
Fix bundle annotations

### PR DESCRIPTION
**Pull request by hatched**
Annotations were not being sent to the environment after deploy of a services from a bundle yaml file.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Get the latest copy of the charm and this branch deployed to a real environment.
```
mkdir -p ~/charms/trusty
cd ~/charms/trusty 
bzr branch lp:~juju-gui/charms/trusty/juju-gui/trunk juju-gui
juju bootstrap local
juju deploy --repository=~/charms local:trusty/juju-gui
juju set juju-gui juju-gui-source="https://github.com/hatched/juju-gui.git fix-bundle-annotations"
```

Drag this bundle to the canvas and deploy it to the environment.
https://gist.github.com/hatched/8e0ac2aba06c942f3c7b

After the services turn yellow refresh the browser. If they are still in the same place then it's good.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2520/
Test PASSed.

File: app/models/models.js [hunk 1]
```diff
@@ -2413,7 +2413,7 @@ YUI.add('juju-models', function(Y) {
                   }
                   return parts.join(':');
                 } else {
-                  return machineIdMap[machineId];
+                  return machineIdMap[machineId] + '';
```
Comment (makyo) [inline app/models/models.js]: Oh, Javascript...

**Comment by makyo** [comment]
:+1: no QA yet

**Comment by makyo** [comment]
QA okay

**Comment by jaycee** [comment]
:+1: thanks @hatched

**Comment by hatched** [comment]
Thanks for the reviews! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**